app/models/game.rb

```diff
@@ -4,4 +4,49 @@ class Game < ApplicationRecord
   belongs_to :player_1, class_name: 'Player', optional: true
   belongs_to :player_2, class_name: 'Player', optional: true
   belongs_to :winning_player, class_name: 'Player', optional: true
+  # Is called after  Base.save on new objects that have not been saved yet (no record exists).
+  # required for testing in console!  without after_create, color would not populate in the correct rows
+  after_create :populate_game!
```

Comment:
😄 

If you need pieces created in a console. Can copy what is done in games controller

```
game = Game.new(name: 'some game')
game.populate_game!
```